CHANGES/7030.bugfix

```diff
@@ -0,0 +1 @@
+Fixes ignoring changes in (custom) repository metadata files on re-sync.
```

Comment:
I think we are trying to use past simple in the changelog messages.

"Fixed re-syncing of custom repository metadata when it was the only change in a repository."?